plugins/rbac-backend/src/helpers/csv.ts

```diff
@@ -0,0 +1,200 @@
+import { Enforcer, FileAdapter, newEnforcer, newModelFromString } from 'casbin';
+import { isEqual } from 'lodash';
+
+import { MODEL } from '../service/permission-model';
+import {
+  validateEntityReference,
+  validatePolicy,
+} from '../service/policies-validation';
+import {
+  checkForDuplicateGroupPolicies,
+  checkForDuplicatePolicies,
+  transformArraytoPolicy,
+  updatePolicies,
+} from './utils';
+
+export const loadFilteredPoliciesFromCSV = async (
+  policyFile: string,
+  enf: Enforcer,
+  entityRef: string,
+  resourceType: string,
+  action: string,
+) => {
+  const fileEnf = await newEnforcer(
```

Comment:
@PatAKnight, maybe we can reuse the same fileAdapter for both methods loadFilteredPoliciesFromCSV and loadFilteredGroupingPoliciesFromCSV to make some performance optimisation?